common/session_test.go

```diff
@@ -34,9 +34,36 @@ import (
 	"github.com/mailru/easyjson"
 	"github.com/stretchr/testify/assert"
 	"github.com/stretchr/testify/require"
+
+	"github.com/grafana/xk6-browser/tests/ws"
 )
 
 func TestSessionCreateSession(t *testing.T) {
+	const (
+		cdpTargetID         = "target_id_0123456789"
+		cdpBrowserContextID = "browser_context_id_0123456789"
+
+		targetAttachedToTargetEvent = `
```

Comment:
We can tackle those problems when we need them. For now, I think it's OK to duplicate things in tests in this way because you can see what's going without referencing other places in code.